starter-code/css/main.css

```diff
@@ -0,0 +1,78 @@
+*{
+  margin: 10px;
+  font-family: Lato;
+  text-align: left;
+
+}
+
+header {
+
+}
+
+h1 a, h4 a {
+  color: rgb(238, 226, 122);
+
+}
+h1 a {
+  font-size: 50px;
+  font-weight: 300;
+  text-decoration: none;
+
+}
+
+h2 {
+  font-weight: 200;
+}
+
+nav a {
+  color: rgb(208, 207, 202);
+  font-size: 20px;
+}
+
+article {
+  max-width: 600%;
```

Comment:
It looks like you might be a bit confused on when to use %. Setting a max-width of 600% means that this article will be 6times as large as the full page or window. I'm pretty sure that's not what you meant to do here. Did you perhaps mean 60% so it only takes up a little over half the window?